src/Port.js

```diff
@@ -0,0 +1,12 @@
+function Port(portName) {
+  this.portName = portName;
+  this.ships = [];
+  this.addShip = function (ship) {
+    this.ships.push(ship);
+  };
+  this.removeShip = function (ship) {
+    this.ships.pop(ship);
+  };
```

Comment:
Is the ship being removed always going to be the last in the array? Would it not be a bit more secure to find the specific element in the list and remove it using an array method like `filter` instead?
```suggestion
  this.removeShip = function(ship) {
    this.ships.pop(ship);
  };
```